### PR TITLE
add training materials from NE BRR 2023 docker training

### PR DESCRIPTION
**Pull request by kapsakcj**
~~Setting as a draft for now, since I'd like to add links to the training materials on the main `/README.md` but wanted to at least get the ball rolling on this PR. Will mark ready for review once that is completed.~~

I realized we never PR'd this back to the main StaPH-B repo.

@kevinlibuit I would like to get your eyes on these training materials before merging. Happy to make edits where you see necessary.

This PR does the following:
- adds the Powerpoint slides (PDF) used in lecture
- markdown document which walks the trainee through the various exercises
- updates the `.gitpod.yml` folder to `git checkout master` instead of `main` which resolves the issue of checking out the repo upon launching the GitPod (see screenshot of GitPod workspace I just spun up)

image

Pull Request (PR) checklist:
- [ ] Include a description of what is in this pull request in this message.
- [ ] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [ ] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [ ] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [ ] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [ ] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [ ] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [ ] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [ ] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

**Comment by kapsakcj** [comment]
I haven't added links to these materials in the main `/README.md` yet, but can do so in a separate PR. It's more important to get these materials added to the main repo

**Comment by kapsakcj** [comment]
@erinyoung FYI I'm using this PR to also add a "Training" section to the main readme with links to the materials included with this PR.

This would be a good location to link to the training you led through APHL: https://learn.aphl.org/learn/course/external/view/elearning/355/intermediate-docker-bioinformatics-workshop

We could include as part of this PR or perhaps add in a separate one. Happy to hear your feedback/thoughts on this.

File: README.md [hunk 1]
```diff
@@ -90,6 +90,10 @@ singularity exec --bind $(pwd)/fastq:/data staphb-bbtools-38.96.simg bbduk.sh in
 
 Further documentation can be found at [docs.sylabs.io](https://docs.sylabs.io/guides/3.1/user-guide/cli.html)
 
+## Training
+
+- Training materials (slides & exercises) from the North East Bioinformatics Regional Resource 2023 workshop **Docker for Public Health Bioinformatics** can be located here, within [`training/NE-BRR-docker-for-PH-bioinformatics-May2023/`](training/NE-BRR-docker-for-PH-bioinformatics-May2023/)
+
```
Comment (erinyoung) [inline README.md:96]: Could you add something like
```
- Video training from APHL 2024 workshop **Intermediate Docker Bioinformatics Workshop** can be located in [APHL's e-learning resources](https://learn.aphl.org/learn/course/external/view/elearning/355/intermediate-docker-bioinformatics-workshop)
```

Comment (kapsakcj) [reply README.md]: Done, here's the rendered version: https://github.com/theiagen/docker-builds/tree/master?tab=readme-ov-file#training

**Comment by erinyoung** [comment]
Closes https://github.com/StaPH-B/docker-builds/issues/1044

**Review by kevinlibuit — APPROVED** [review]
Looks good! Thanks for adding this, @kapsakcj.

One quick note on something I found interesting: I like that you've updated it to `checkout master` to be more specific, though it seems that the current .gitpod.yml also sets the environment to master by default and may just ignore that `checkout main` line as there is no `main` branch to checkout.

**Comment by kapsakcj** [comment]
You are correct, it's smart enough to know which branch to check out, but the change in this PR just eliminates the error or warning (can't remember which) that occurs upon starting the gitpod environment

**Pull request merged**